### DISCUSSION TITLE
evaluation metrics

### DISCUSSION THREAD

**Discussion by Arwa-97** [keras-team/keras-cv#2193] [Q&A] [answered]
hi,
how can I calculate the KID or FID using the Fine-tuning Stable Diffusion sample code?
or where should I modify the code to make these calculation 

thanks

**Comment by divyashreepathihalli** [comment, marked as answer]
@Arwa-97 here is an example of implementing [KID in keras 3](https://keras.io/examples/generative/ddim/#:~:text=class%20KID(keras.metrics.Metric)%3A) - you would add this as a `keras.metrics.Metric` subclass and then specify this as a metric to `model.compile(...)`
